src/components/ProviderCard/ProviderCard.stories.js

```diff
@@ -0,0 +1,24 @@
+import ProviderCard from './ProviderCard';
+import DIDLogo from '../../assets/logos/didperu.svg';
+
+export default {
+  title: 'Pages/Providers/ProviderCard',
+  component: ProviderCard,
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
+const Template = (args) => ({
```

Comment:
What about naming it with something related to the component?

Reply:
"Template" is not visible for users on storybook, it's just used to created multiple examples with different props, as you can see on "Default"

```
export const Default = Template.bind({});
// More on args: https://storybook.js.org/docs/vue/writing-stories/args
Default.args = {
  image_url: DIDLogo,
  id: '1',
};
```

The idea is that I can create another example without add all the settings of Template:

```
export const CasaRosello = Template.bind({});
// More on args: https://storybook.js.org/docs/vue/writing-stories/args
Default.args = {
  image_url: CasaRosello,
  id: '3',
};
```

Then the user will see two options:

- Default
- CasaRosello

---

But I think it's a good theme of conversation, i.e, in this case I have just one example on Storybook, do you think if it's just one example on storybook is better to add the story directly instead of create a Template?

Reply:
What about "ProviderCardTemplate" or similar for the template?

Reply:
Hmmm ok I think it's ok for the template, maybe it helps the developer to find quickly the story for that component